src/main/storm/mesos/MesosSupervisor.java

```diff
@@ -112,7 +108,7 @@ public String getSupervisorId() {
 
   @Override
   public String getAssignmentId() {
-    return _assignmentId;
+    return MesosCommon.hostFromAssignmentId(_assignmentId, MesosCommon.getWorkerPrefixDelimiter(_conf));
```

Comment:
I still do not understand *why* @maverick2202 wants to change this assignmentId stuff.  Please explain *why* we are adding this extra logic.